app/views/layouts/application.html.slim

```diff
@@ -18,28 +18,29 @@ html lang="#{I18n.locale || I18n.default_locale}" data-default-locale="#{I18n.de
     meta name='current-user' content=current_user&.to_page_context&.to_json
   body.d-flex.flex-column.h-100
     #content
-      nav.navbar.bg-light.navbar-expand-lg.mb-0 role='navigation' data-bs-theme="light"
-        .container
-          = link_to home_index_path, class: "navbar-brand"
-            i.fa-solid.fa-anchor.me-2#logo
-            = application_name
-          button.navbar-toggler data-bs-target='#navbar-collapse' data-bs-toggle='collapse' type='button' aria-expanded='false' aria-label='Toggle navigation' aria-controls='navbar-collapse'
-            span.navbar-toggler-icon
-          #navbar-collapse.collapse.navbar-collapse
-            = render('navigation')
-            ul.nav.navbar-nav.ms-auto
-              = render('session')
+      header.py-3
```

Comment:
To me, it seems like `py-3` is neither enough nor probably the best, generic way to fix this problem. As mentioned above from the flash message, let's ensure that this `header` has the correct height, so that we don't need to mingle with it.

<img width="629" alt="Bildschirmfoto 2023-12-06 um 23 36 50" src="https://github.com/openHPI/codeharbor/assets/7300329/2bd09f7c-a0bc-4c47-ae7d-12cb55f63674">

Reply:
Thanks for working on the header again. Unfortunately, I still feel we are misusing some of the bootstrap utility classes (here: `mt-4`, `mt-5`) to provide a work-around.

Would you please investigate once again on how to solve this issue without a workaround or without a fixed height? 

The `header` tag still has no height:

![Bildschirmfoto 2023-12-12 um 15 52 58](https://github.com/openHPI/codeharbor/assets/7300329/6d3d8393-f006-4ae3-8d00-040d2432cce5)

Hence, the following element `main` is (partly) hidden by the navbar:

![Bildschirmfoto 2023-12-12 um 15 52 39](https://github.com/openHPI/codeharbor/assets/7300329/62c20205-6627-4111-ba6a-98345fccbc0b)

Here, you are using the `mt-4` class to add some margin, making all content visible. Similar, the `mt-5` at the `body` tag really doesn't feel right.

For example, consider openHPI: Here, the actual content doesn't require any padding, margins, ... since the navbar height is set correctly:

![Bildschirmfoto 2023-12-12 um 15 55 24](https://github.com/openHPI/codeharbor/assets/7300329/9774761e-bb9a-40a5-b146-cd068ac00c28)

Reply:
Yes, I have not finished the header yet. I am trying other solutions as of now. Will push the change again sometime today!

Reply:
After tinkering, unfortunately, it is impossible to stack the element under `fixed-top` navbar in [bootstrap](https://getbootstrap.com/docs/5.3/components/navbar/#placement). 
What we can do is, remove those unnecessary mt-* classes and put a custom padding-top (let's say 60px) to give the element below the navbar some space.
Still, it gives `header` tag 0 height because it is just a wrapper for the .navbar fixed position. But afaik that is normal (or we could just erase the `header`/`main` tag completely since it is actually optional. OpenHPI also does not use these tags).

Reply:
The answer is quite simple actually! Lucky I tried to "really" understand how sticky and fixed works in bootstrap, and applied it to the right element!

Reply:
Awesome, thanks! Indeed, this looks much cleaner and feels more like a permanent solution. Thank you!